app/app.component.ts

```diff
@@ -12,7 +12,11 @@ import {About} from './sections/about/about.component';
   selector: 'gapminder-app',
   encapsulation: ViewEncapsulation.None,
   directives: [Header, Footer, ...ROUTER_DIRECTIVES],
-  styles: [ <string> require('./main.styl')],
+  styles: [
+    <string> require('./main.styl'),
+    <string> require('./fonts.styl'),
+    <string> require('./variables.styl')
```

Comment:
ok, understood. I just don't like mess inside the root app folder. let's leave it and we'll see how it'll be looked like.